protocol/src/main/java/org/fidoalliance/fdo/protocol/db/To2BlobSupplier.java

```diff
@@ -35,7 +35,7 @@ public To2AddressEntries get() throws IOException {
         onboardConfig.setReplacementRvInfo(null);
 
         //3=http or 5=https
-        final String defaultBob = "[[\"127.0.0.1\",\"localhost\",%s,3]]";
+        final String defaultBob = "[[null,\"host.docker.internal\",%s,3]]";
```

Comment:
if it's null; then client-sdk fails to parse TO2AddrEntry properly.

Reply:
++ An exception occurred processing Appender RollingFile java.lang.ClassCastException: class org.h2.value.lob.LobDataFetchOnDemand cannot be cast to class org.h2.value.lob.LobDataDatabase (org.h2.value.lob.LobDataFetchOnDemand and org.h2.value.lob.LobDataDatabase are in unnamed module of loader 'app')
pri-fdo-aio    |        at org.h2.value.ValueBlob.getSQL(ValueBlob)